src/vsr/replica.zig

```diff
@@ -2954,56 +3099,21 @@ pub fn ReplicaType(
                 .timestamp = if (command == .do_view_change) self.log_view else 0,
                 .op = self.op,
                 // See the comment in `on_do_view_change()` for why `commit_min` is crucial:
-                .commit = if (command == .do_view_change) self.commit_min else self.commit_max,
+                .commit = self.commit_min,
```

Comment:
Just a simplification. For SV, we are the primary, so `commit_max == commit_min` anyway.